contribute.md

```diff
@@ -70,9 +70,9 @@ If you need to reproduce the site locally and/or make extensive changes, you can
 
 ### Forking this Repository
 
-First check to see if you already have a fork of this repository:
+First check to see if you already have a fork of this repository at:
 
-[https://github.com/<your username>/design][]
+https://github.com/{your github username}/design
```

Comment:
The `{}` typically denotes "pick one of", e.g. `{up, down}`, whereas `<thing>` means mandatory replace. This in the style of USAGE or man pages. So, I think that it should remain inside `<>`. Also, if you don't make a link out of it with the `[name](url)` then it will not link. I think instead it should just be in a pre using the ``` `` ```.